victorops-twilio.js

```diff
@@ -1107,11 +1111,15 @@ function postToVictorOps (event, context, payload) {
       }
     )
     .then(response => {
-      resolve('');
+      const twiml = new Twilio.twiml.VoiceResponse();
+      twiml.say(
+        {voice},
+        `${messages.otherPartyDisconnect} ${messages.goodbye}`
+      );
+      resolve(DialCallStatus == 'completed' && TranscriptionStatus !== 'failed' ? twiml : '');
```

Comment:
resolve(DialCallStatus === 'completed' && TranscriptionStatus !== 'failed' ? twiml : '');

Reply:
Using === prevents recovery alerts when the callee hangs up.